src/main/kotlin/model/lotto/LottoTicket.kt

```diff
@@ -1,41 +1,13 @@
 package model.lotto
 
-enum class TicketType {
-    Manual,
-    Auto,
-}
-
-data class LottoTicket(
-    val numbers: LottoNumbers,
-    val type: TicketType,
-) {
+class LottoTicket(val games: List<LottoGame>) {
     companion object {
-        fun of(
-            numbers: LottoNumbers,
-            type: TicketType,
-        ): LottoTicket {
-            return LottoTicket(numbers, type)
+        fun of(games: List<LottoGame>): LottoTicket {
+            return LottoTicket(games)
         }
     }
 
```

Comment:
```suggestion
        fun createWithGames(games: List<LottoGame>): List<LottoTicket> {
            return games.chunked(5) { LottoTicket.of(it) }
        }
```

이런식으로 chunked(`개수`) 라는 녀석을 이용해보는 건 어떨까요?

Reply:
코틀린에서 제공하는 유틸함수가 많군요 ㅎㅎ
덕분에 배웠습니다!